### ISSUE TITLE
Multinode nginx reverse proxy missing RemoteIpValve node in conf/server.xml?

### ISSUE THREAD

**Issue by vong0** [JetBrains/teamcity-docker-samples#6]
According to the [docs](https://www.jetbrains.com/help/teamcity/how-to.html#%22RemoteIpValve%22+Approach) ,
when the following headers in nginx are set 
`X-Forwarded-Proto, X-Forwarded-Port`, you should add the following into the Tomcat main <Host> node of the conf\server.xml file.
```
<Valve
   className="org.apache.catalina.valves.RemoteIpValve"
   remoteIpHeader="x-forwarded-for"
   protocolHeader="x-forwarded-proto"
   portHeader="x-forwarded-port"
   />
```

But I noticed that you did not make this change, even though you set those headers in nginx. Is there a reason you did not? Is this a bug? What will happen if this node is not added?

**Comment by kir** [comment]
No, there is no reason I did not do it. Most features in TeamCity should work without setting this section, AFAIK

Have you noticed any problems due to the absence of this section?

**Comment by vong0** [comment]
No. I havent noticed any issues with the absense of this section. I was just wondering why it missing when the docs said to include it.
